GitUI/CommandsDialogs/FormRebase.cs

```diff
@@ -123,8 +129,11 @@ protected override void OnShown(EventArgs e)
             EnableButtons();
 
             // Honor the rebase.autosquash configuration.
-            string autosquashSetting = Module.GetEffectiveSetting("rebase.autosquash");
-            chkAutosquash.Checked = autosquashSetting.Trim().ToLower() == "true";
+            chkAutosquash.Checked = Module.GetEffectiveSetting<bool>("rebase.autosquash") is true;
+            if (Module.GitVersion.SupportUpdateRefs && Module.GetEffectiveSetting<bool>("rebase.updateRefs") is true)
```

Comment:
If `rebase.updateRefs` is unset that is it presumed `false`?
```suggestion
            if (Module.GitVersion.SupportUpdateRefs && Module.GetEffectiveSetting<bool>("rebase.updateRefs"))
```

Reply:
No, it's not building...

Reply:
... because nullable